e2e/tests/cookie-banner.spec.ts

```diff
@@ -1,15 +1,22 @@
-import { expect, test } from '@playwright/test'
+import { expect, Locator, test as base } from '@playwright/test'
 
+const test = base.extend<{ cookieBanner: Locator }>({
+  cookieBanner: async ({ page }, use) => {
+    const cookieBanner = page.getByRole('region', { name: 'Cookies on the MOD.UK Design System' })
+    await use(cookieBanner)
+  },
+})
+const gaCookieRegex = /_ga=(\w|\.)+; _ga_[A-Z0-9]{10}=(\w|\.)+/
```

Comment:
Doesn't look like this solves the problem of the order of the cookies not being important. If you don't like multiple expectations, going back to how it was but sorting the array first should do it I think?